generator/overrides/education/enums/errorCodes.ts

```diff
@@ -0,0 +1,13 @@
+export enum ErrorCodes {
```

Comment:
why not in the `template` folder?
they r equal for each app

Reply:
yes, they are equal. But we need ErrorCodes in `generator/overrides/education/pages/issuer/credential-form/index.tsx` for that had to create enum in overrides

Reply:
if u r added it to `template/enums/errorCodes` it will auto-generate it everywhere